docker/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.6'
 services:
   eva-catalog:
-    image: ${SKYNET_APPLICATION_EVA_CATALOG:-drydock.workiva.net/workiva/eva-catalog:latest-release}
+    image: workivadocker/eva-catalog:latest-release
```

Comment:
This should be `${SKYNET_APPLICATION_EVA_CATALOG:-workivadocker/eva-catalog:latest-release}` so that Skynet will use the build from the PR.